plutus-core/untyped-plutus-core/test/Evaluation/Builtins/Common.hs

```diff
@@ -39,7 +39,7 @@ typecheckEvaluateCek
     => MachineParameters CekMachineCosts CekValue uni fun
     -> TPLC.Term TyName Name uni fun ()
     -> m (EvaluationResult (UPLC.Term Name uni fun ()), [String])
-typecheckEvaluateCek = typecheckAnd unsafeEvaluateCek
+typecheckEvaluateCek = typecheckAnd $ unsafeEvaluateCek False
```

Comment:
`unsafeEvaluateCek` is meant for evaluation only, so unless you want to show the logs sometimes, just hardcode that `False` right inside `unsafeEvaluateCek`.